### PR TITLE
Fix customElements.upgrade() tests for <template>

### PR DESCRIPTION
**Pull request by domenic**
See https://github.com/w3c/web-platform-tests/pull/9869#issuecomment-378002725. @bicknellr please take a look! Thank you for finding this.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26733)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/361307529)

File: custom-elements/custom-element-registry/upgrade.html [hunk 1]
```diff
@@ -145,10 +145,13 @@
...
-  assert_true(el4 instanceof Element4, "element 4 must now be upgraded");
-  assert_true(el5 instanceof Element5, "element 5 must now be upgraded");
+  // Template contents owner documents don't have a browsing context, so
+  // https://html.spec.whatwg.org/multipage/custom-elements.html#look-up-a-custom-element-definition does not find any
+  // custom element definition.
+  assert_true(el1 instanceof Element1, "element 1 must still not be upgraded after upgrading the template contents");
+  assert_true(el2 instanceof Element2, "element 2 must still not be upgraded after upgrading the template contents");
+  assert_true(el3 instanceof Element3, "element 3 must still not be upgraded after upgrading the template contents");
+  assert_true(el4 instanceof Element4, "element 4 must still not be upgraded after upgrading the template contents");
+  assert_true(el5 instanceof Element5, "element 5 must still not be upgraded after upgrading the template contents");
```
Comment (bicknellr) [inline custom-elements/custom-element-registry/upgrade.html]: `assert_true` -> `assert_false` for these lines

Comment (domenic) [reply custom-elements/custom-element-registry/upgrade.html]: goodness -_-

**Comment by bicknellr** [comment]
:+1:

**Review by tkent-google — APPROVED** [review]

**Pull request merged**